test/suites/dev-tanssi-relay/external-validators-rewards/test_external_validator_reward_candidates.ts

```diff
@@ -0,0 +1,67 @@
+import "@tanssi/api-augment";
+import { describeSuite, customDevRpcRequest, expect, beforeAll } from "@moonwall/cli";
+import { ApiPromise, Keyring } from "@polkadot/api";
+import { jumpToSession } from "util/block";
+
+describeSuite({
+    id: "DTR1601",
+    title: "Paras inherent tests",
+    foundationMethods: "dev",
+
+    testCases: ({ it, context }) => {
+        let polkadotJs: ApiPromise;
+
+        beforeAll(async () => {
+            polkadotJs = context.polkadotJs();
+        });
+
+        it({
+            id: "E01",
+            title: "para candidates should trigger reward info",
+            test: async function () {
+                const keyring = new Keyring({ type: "sr25519" });
+                const aliceStash = keyring.addFromUri("//Alice//stash");
+                await context.createBlock();
+                // Send RPC call to enable para inherent candidate generation
+                await customDevRpcRequest("mock_enableParaInherentCandidate", []);
+                // Since collators are not assigned until session 2, we need to go till session 2 to actually see heads being injected
+                await jumpToSession(context, 3);
+                await context.createBlock();
+
+                // we are still in era 0
+                const validatorRewards = await context
+                    .polkadotJs()
+                    .query.externalValidatorsRewards.rewardPointsForEra(0);
+                const totalRewards = validatorRewards.total.toBigInt();
+
+                expect(totalRewards).to.be.greaterThan(0n);
```

Comment:
we should make sure here that rewards are higher than what the validator gets for producing a block. Otherwise we are not testing specifically the rewards given by candidates.

For that you probably need to fetch the previous blocks reward accumulation points, the current one, and the difference should be higher than 20

Reply:
*fixed*